demo/save_file_no_output/run.py

```diff
@@ -0,0 +1,16 @@
+import random
+import string
+import gradio as gr 
+
+def save_image_random_name(image):
+    random_string = ''.join(random.choices(string.ascii_letters, k=20)) + '.png'
+    image.save(random_string)
+    print(f"Saved image to {random_string}!")
+
+demo = gr.Interface(
+    fn=save_image_random_name, 
+    inputs=gr.Image(type="pil"), 
+    outputs=None,
+)
+if __name__ == "__main__":
+    demo.launch()
```

Comment:
There's no feedback at all to the user in this demo. Maybe that's the point, but it can get a little confusing. I think the image should at least disappear. Could be done with blocks and a clear event I suppose, maybe not worth it so will leave it up to you. But try it out here if you haven't to see what I mean, just feels like button isn't doing anything: https://huggingface.co/spaces/gradio/save_file_no_output

Reply:
That's the point though. If we rewrote it with Blocks, it would no longer be about the kinds of Interfaces